src/content/documentation/develop/extensions-and-the-add-on-id.md

```diff
@@ -114,7 +116,7 @@ You can do the same thing if you are updating from an older add-on type, such as
   - [`pkcs11`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pkcs11 'The pkcs11 API enables an extension to enumerate PKCS #11 security modules and to make them accessible to the browser as sources of keys and certificates.')
   - [`runtime.onMessageExternal`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/runtime/onMessageExternal "This API can't be used in a content script.")
   - [`runtime.onConnectExternal`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/runtime/onConnectExternal 'Fired when an extension receives a connection request from a different extension.')
-- Using Firefox for Android. See [`browser_specific_settings` in manifest.json](https://developer.mozilla.org/Add-ons/WebExtensions/manifest.json/browser_specific_settings).
+- Using Firefox before version 48. See [`browser_specific_settings` in manifest.json](https://developer.mozilla.org/Add-ons/WebExtensions/manifest.json/browser_specific_settings).
```

Comment:
"Firefox for Android" was not correct. I think that it was here because Fennec extension development usually worked with xpi files since it didn't have `about:debugging`, and consequently the "Firefox for Android requires an ID" misconception was created.

I've removed the line though. It's not worth mentioning here.

Reply:
> I've removed the line though. It's not worth mentioning here.
that was my preference too, thanks